package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A React component for swipeable views",
   "main": "lib/index.js",
   "scripts": {
+    "lint": "eslint --ext .js .jsx .",
```

Comment:
It's linting the lib folder but shouldn't (e.g. /home/travis/build/oliviertassinari/react-swipeable-views/lib/index.js)